src/components/App/AccountBase.js

```diff
@@ -0,0 +1,37 @@
+import React, { PureComponent } from 'react';
+import { AssignmentReturn, AssignmentReturned, SaveAlt } from '@material-ui/icons';
+import { Button, Avatar, Box } from '@material-ui/core';
+import { login, logout, saveUser } from '../../actions/AppStoreActions';
+
+class AccountBase extends PureComponent {
+    render() {
+        const user = this.props.user;
+        return (
+            <>
+                {user ? (
+                    <>
+                        <Avatar
+                            alt={user.passport.user.name}
+                            src={user.passport.user.picture}
+                            style={{ marginRight: 5, width: 25, height: 25 }}
+                        ></Avatar>
```

Comment:
I think this can be self closing
```suggestion
                        />
```